src/Diff/index.tsx

```diff
@@ -22,6 +22,9 @@ export interface DiffProps {
     viewType?: ViewType;
     gutterType?: GutterType;
     generateAnchorID?: (change: ChangeData) => string | undefined;
+    generateLineClassName?: (
```

Comment:
This type is not strict, we should narrow it to either one or two change data, infer it from `viewType` prop.

You may directly change it, or I will change it after I merged this PR.